src/com/gui/Configuration.java

```diff
@@ -0,0 +1,119 @@
+package com.gui;
+
+import com.utils.writers.PropertiesWriter;
+
+import java.util.Scanner;
+
+/**
+ * This class represent the Configuration console interface
+ */
+public class Configuration extends Console {
+    private PropertiesWriter propertiesWriter = new PropertiesWriter();
+
+    public void start() {
+        Boolean exit = false;
+        do {
+            display("------Configuration Settings------" + "\n");
+            display("1.Output type");
+            display("2.Level");
+            display("3.Algorithm");
+            display("4.Interface");
+            display("5.Exit");
+            display("Select an option:");
+            input = new Scanner(System.in);
+            int option = input.nextInt();
+            switch (option) {
+                case 1:
+                    enterOutputType();
```

Comment:
You mean with something like this:
   protected static final Map<Integer, Method> MENU_OPTIONS =  new HashMap<>();
    {
        MENU_OPTIONS.put( 1, Configuration.class.getMethod("enterOutputType"));
        MENU_OPTIONS.put( 2, Configuration.class.getMethod("selectLevel"));
        MENU_OPTIONS.put( 3, Configuration.class.getMethod("selectAlgorithm"));
        MENU_OPTIONS.put( 4, Configuration.class.getMethod("enterCharacter"));
        MENU_OPTIONS.put( 5, Configuration.class.getMethod("enterOutputType"));
    }